src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -102,7 +104,13 @@ public boolean hasPerson(Person person) {
 
     @Override
     public void deletePerson(Person target) {
+        requireNonNull(target);
+
+        // Remove the person from the address book
         addressBook.removePerson(target);
+
+        // Remove the person from all groups
+        removePersonFromAllGroups(target);
```

Comment:
great job adhering to SLAP, by creating a removePersonFromAllGroups method, you implemented abstraction really well!